_kbarticles/2021-05-24-secureauth-issues.md

```diff
@@ -20,3 +22,15 @@ Things you can check:
 * You can try clearing all cache and cookies, restart your browser, and retry logging in.
 
 If these steps don't work, [GSA also has a tool that can clear these items](https://secureauth.gsa.gov/secureauth2/cleancert.aspx). Note - if you are a Mac user, this site will only work in Safari.
+
+## If you are using the cloud.gov identity provider (IDP)
+
+For users that log in with the cloud.gov IDP, if a user hasn't logged in in 90 days their account will be deactivated for security purposes. You should receive an email notice when a deactivation is imminent. 
```

Comment:
When users end up in this situation, when they try and login (even with a password reset), they'll experience SAML errors as well.  These can also be caused by stale browser cookies and cache, so I wonder if we should make mention of this.  Perhaps like this?

```suggestion
For users that log in with the cloud.gov IDP, if a user hasn't logged in in 90 days their account will be deactivated for security purposes. You should receive an email notice when a deactivation is imminent, and you may experience receiving SAML errors when logging in (even after a password reset).
```